README.md

```diff
@@ -13,7 +13,7 @@ sysdig
 
 Where to start?
 ---
-If this is your first time hearing about sysdig, we recommend you [start with the website](http://www.sysdig.org).  
```

Comment:
I think we should keep sysdig.org here, In the past there was a dedicated site for open source offerings. It seems to redirect to sysdig.com at the moment but this way it will point to the right site once if it comes back.